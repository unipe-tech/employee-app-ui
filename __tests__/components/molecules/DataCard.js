import { render, fireEvent, waitFor } from "@testing-library/react-native";
import { describe, expect, test } from "@jest/globals";
import DataCard from "../../../components/molecules/DataCard";

describe("DataCard", () => {
  test("renders UI correctly", () => {
    const tree = render(<DataCard data="Testing Data" />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
