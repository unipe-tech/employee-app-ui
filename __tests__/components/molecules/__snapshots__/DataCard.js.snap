// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DataCard renders UI correctly 1`] = `
<RCTScrollView
  showsVerticalScrollIndicator={false}
  style={
    Object {
      "height": "63%",
      "marginTop": "1.5%",
    }
  }
>
  <View />
</RCTScrollView>
`;
